data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-            [objective]
-                description= _ "Death of Dacyn"
-                condition=lose
-            [/objective]
-
-            {TURNS_RUN_OUT}
-
-            [gold_carryover]
-                bonus=yes
-                carryover_percentage=40
-            [/gold_carryover]
-
-#ifndef HARD
-            [note]
-                description= _ "Get moving quickly."
-            [/note]
-#endif
-        [/objectives]
-
-        [recall]
-            id=Dacyn
-        [/recall]
-    [/event]
-
-    [event]
-        name=start
-
-        [message]
-            speaker=Gweddry
-            message= _ "Where are we? I can’t see where we are going."
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "This is an old escape route for the outpost. Unfortunately these caves are currently inhabited by trolls, who are blocking our path through the tunnel."
-        [/message]
-
-        # Lift shroud around the amulet
-        [scroll_to]
-            x,y=12,2
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=12,2
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        # Path leading through the troll cave
-        [scroll_to]
-            x,y=15,7
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=15,7
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=22,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=22,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=28,6
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=28,6
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=34,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=34,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [message]
-            speaker=Dacyn
-            message= _ "We must make haste to escape before the undead catch up."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "Wait, before we go anywhere — who were those undead?"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I will explain later. For now, suffice to say they are much too powerful for us; our only hope is to head north."
-        [/message]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=29,10
+#
+# Simple and easy, especially compared to S01.
+#
+# This is beatable without any combat or recruiting - just run straight through the dwarfish tunnels, and wade through the river.
```

Comment:
my habit is always to recruit anyways though

Reply:
Yeah, and that's definitely both the funnest and mechanically optimal way to play.  Comment is just pointing out that you don't have to if you really don't want to (that's also the only way to get the achievement on Hard IIRC).